clients/admin-ui/src/features/privacy-requests/configuration/MailgunEmailConfiguration.tsx

```diff
@@ -0,0 +1,223 @@
+import { Box, Button, Divider, Heading, Stack } from "@fidesui/react";
+import { Form, Formik } from "formik";
+import { useState } from "react";
+
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage } from "~/features/common/helpers";
+import { useAlert } from "~/features/common/hooks";
+import {
+  useCreateMessagingConfigurationMutation,
+  useCreateMessagingConfigurationSecretsMutation,
+  // useGetMessagingConfigurationDetailsQuery,
+} from "~/features/privacy-requests/privacy-requests.slice";
+
+const MailgunEmailConfiguration = () => {
+  const { errorAlert, successAlert } = useAlert();
+  const [configurationStep, setConfigurationStep] = useState("");
+  // const { data: messagingDetails } = useGetMessagingConfigurationDetailsQuery({
+  //   type: "mailgun",
+  // });
+  const [createMessagingConfiguration] =
+    useCreateMessagingConfigurationMutation();
+  const [createMessagingConfigurationSecrets] =
+    useCreateMessagingConfigurationSecretsMutation();
+
+  const handleMailgunConfiguration = async (value: { domain: string }) => {
+    const payload = await createMessagingConfiguration({
```

Comment:
Scanning the codebase, it looks like `payload` is/was used to refer to the body of a response after it's been [unwrapped](https://github.com/ethyca/fides/blob/38d932aa9e0f3931addb3aae7c3b39afd82ea026/clients/admin-ui/src/features/datastore-connections/add-connection/DatasetConfiguration.tsx#L77)*. We tend to go with `result` when referring the object that safely wraps the success/error data (that's what RTK calls it.)

\* The `payload = ...unwrap()` code that shows up when you use intellisense on a mutation function is unsafe (it throws exceptions) and unnecessary when we have our error message helpers 💁